cmd/telepresence/main.go

```diff
@@ -30,6 +33,25 @@ func main() {
 
 	if err := cmd.ExecuteContext(ctx); err != nil {
```

Comment:
There are some situations where it doesn't make sense to print the logs:
1. This is not the  CLI process (it's the `connector` or `daemon` executing in the background).
2. This is the CLI process, but the error is a usage error (not logged).
3. The error is expected, but not anything that has info in the logs. E.g. user attempts to logout but are not logged in.

Reply:
Is there a way to detect if you are the connector or daemon? I added this here because @LukeShu recommended this as the place to start, but I'm happy to make this more specific if that's possible.